internal/pkg/manifest/lb_web_svc.go

```diff
@@ -55,6 +55,7 @@ type LoadBalancedWebServiceConfig struct {
 	Network          NetworkConfig             `yaml:"network"`
 	PublishConfig    PublishConfig             `yaml:"publish"`
 	TaskDefOverrides []OverrideRule            `yaml:"taskdef_overrides"`
+	NLBConfig        bool                      // TODO: placeholder field that will be expanded to NLBConfig struct.
```

Comment:
nit: let's not include dead code if the field isn't used yet, we can add NLBConfig once the struct is ready

Reply:
I will cherry-pick my manifest changes to this PR